test/unit/src/common/components/user-avatar/t_user-avatar.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import UserAvatar, { UserAvatarView } from '../../../../../../src/common/components/user-avatar';
```

Comment:
Testing both container and view components.

As suggested in redux docs (http://redux.js.org/docs/recipes/WritingTests.html#connected-components) we may not necessarily need `View` suffix on the name, but maybe it does things a bit more clear?